### PR TITLE
man page for pyp2rpm

### PR DESCRIPTION
**Pull request by rsuste**

### CODE REVIEW

**Comment by hroncok** [comment]
Thanks for contributing.

Did you just write it in groff syntax or used some tool to generate it? It would be better to generate it, so it is not out of date soon.

**Comment by rsuste** [comment]
I wrote it in groff. Do you want it remake to Markdown or something other?
Dne 25.5.2016 12:03 napsal uživatel "Miro Hrončok" <notifications@github.com

**Comment by hroncok** [comment]
Not really, that would keep the problem. I was thinking generating it from the --help or something like that.

**Comment by xsuchy** [comment]
@hroncok I advised @RadimS97 to not generate it from --help as he wrote there more information than is available in --help output. E.g. Example section.

**Comment by hroncok** [comment]
And that's a good thing, but I'm afraid that the manpage will get outdated, so I'm thinking if there is a way to keep it up to date automatically somehow, but maybe I'm just overthinking it.

File: pyp2rpm.1 [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+.TP
+.B pyp2rpm ~/rpmbuild/SOURCES/python_package_name.tar.gz
+Pick up package from the local filesystem.
+
+ 
+
+	
+
+.SH LANGUAGE
+PYTHON
```
Comment (hroncok) [inline pyp2rpm.1:88]: Is this a common section? I don't see it's purpose.

File: pyp2rpm.1 [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+.SH SYNOPSIS
+.B pyp2rpm
+[-h] [-v VERSION] [-d SAVE_DIR] [-r RPM_NAME] [-t TEMPLATE]
+[-o DISTRO] [-b BASE_PYTHON] [-p PYTHON_VERSION] [--srpm] [--proxy PROXY]
+.B PACKAGE
+
+
+.SH ARGUMENTS
+.B PACKAGE
```
Comment (hroncok) [inline pyp2rpm.1:15]: Maybe use .TP?

**Comment by xsuchy** [comment]
> but maybe I'm just overthinking it.

:) Most of the projects have man page as separate file. And yes, it is one more thing to think about when you add feature/option that you need to write up documentation. But that is life and it happen when some project grown up.

**Comment by mcyprian** [comment]
@hroncok Do you have some more comments to this? Can I merge this PR now?

**Comment by hroncok** [comment]
I've added 2 inline comments, no response, feel free to consider them irrelevant and marge anyway, or change it after the merge.

**Pull request merged**